include/swift/AST/Attr.def

```diff
@@ -254,33 +261,38 @@ DECL_ATTR(_specialize, Specialize,
   33)
 SIMPLE_DECL_ATTR(objcMembers, ObjCMembers,
   OnClass |
-  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
+  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIBreakingToRemove,
```

Comment:
I think technically this is OK, because it infers @objc which is API breaking to remove.

If the digester doesn't look at inferred @objc attributes, then this one is correct, but a few above are like @IBAction are wrong; if you remove one (and don't consider that you've removed the inferred @objc part) then you've broken ABI because the method possibly is no longer @objc.

Reply:
It does look at inferred `@objc` attributes, but `@objcMembers` can go on generic things too, right?